cluster/src/main/scala/org/apache/spark/memory/SnappyUnifiedMemoryManager.scala

```diff
@@ -397,16 +406,16 @@ private object SnappyUnifiedMemoryManager extends Logging{
     val minSystemMemory = (reservedMemory * 1.5).ceil.toLong
     if (systemMemory < minSystemMemory) {
       throw new IllegalArgumentException(s"System memory $systemMemory must " +
-          s"be at least $minSystemMemory. Please increase heap size using the --driver-memory " +
-          s"option or spark.driver.memory in Spark configuration.")
+        s"be at least $minSystemMemory. Please increase heap size using the --driver-memory " +
+        s"option or spark.driver.memory in Spark configuration.")
```

Comment:
@rishitesh Please take note of the messages. We need to correct these to mention "heap-size/memory-size" for cluster mode and the other properties in local mode.